process/lifecycle.md

```diff
@@ -15,7 +15,7 @@ This document provides for three lifecycle stages for contributed projects:
 * Adopted; and
 * Archived
 
-All projects must meet the Incubation stage requirements and following a successful Graduation Review, progress to Adopted. It is possible that some projects may be approved as Incubation and pass a Graduation Review at the same time to advance directly to Adopted. A project that does not pass a Graduation Review remains at the Incubation stage unless the TAC votes to transition the project to Archive status.
+All projects must meet the Incubation stage requirements and following a successful Graduation Review, progress to Adopted. It is possible that some projects may be approved as Incubation and pass a Graduation Review at the same time to advance directly to Adopted. A project that does not pass a Graduation Review remains at the Incubation stage unless the TAC votes to transition the project to Archive status.  Alternatively, the TAC or the project TSC may vote to exit the lifecycle under specific terms for transfer of trademarks and other key assets.
```

Comment:
What's intended by the "specific terms" clause for exiting the lifecycle?

Reply:
I was attempting to capture the fact that specifics of transfer arrangements and so on would need to be part of any exit vote.  cf Brexit.

Reply:
Looks good to me.